AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                general.idle_wait_frames(1)
+                Report.result(
+                    Tests.enable_shadows,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Enable shadow')) is True)
+
+                # 14. Edit the Shadows Bias parameter.
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Shadows Bias'), 100)
+                Report.result(
+                    Tests.edit_shadow_bias,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Shadows Bias')) == 100)
+
+                # 15. Edit the Normal shadow bias parameter.
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Normal shadow bias'), 10)
+                Report.result(
+                    Tests.edit_normal_bias,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Normal shadow bias')) == 10)
+
+                # 16. Set the Shadowmap size.
+                # for shadowmap_size in SHADOWMAP_SIZE.keys():
+                #     light_component.set_component_property_value(
+                #         AtomComponentProperties.light('Shadowmap size'), SHADOWMAP_SIZE[shadowmap_size])
+                #     current_shadowmap_size = light_component.get_component_property_value(
+                #         AtomComponentProperties.light('Shadowmap size'))
+                #     test_shadowmap_size = (
+                #         f"Shadowmap size for Light type: {light_type} set to {shadowmap_size}.",
+                #         f"P1: Shadowmap size for Light type: {light_type} failed to be set to {shadowmap_size}.")
+                #     Report.result(test_shadowmap_size, current_shadowmap_size == SHADOWMAP_SIZE[shadowmap_size])
+
+                # Shadow filter method tests.
+                # 17. Set the Shadow filter method.
+                for filter_method in SHADOW_FILTER_METHOD.keys():
+                    light_component.set_component_property_value(
+                        AtomComponentProperties.light('Shadow filter method'), SHADOW_FILTER_METHOD[filter_method])
+                    current_filter_method = light_component.get_component_property_value(
+                        AtomComponentProperties.light('Shadow filter method'))
+                    test_shadow_filter_method = (
+                        f"Shadow filter method set to {filter_method}",
+                        f"P1: Shadow filter method set to {filter_method}")
+                    Report.result(test_shadow_filter_method,
+                                  current_filter_method == SHADOW_FILTER_METHOD[filter_method])
+
+                    # 18. Edit the Filtering sample count parameter.
+                    if current_filter_method in (1, 3):
```

Comment:
if filter_method in ['whatever 1 is', 'whatever 3 is']
debugging or maintaining this requires keeping track of what 1, 3 means. if the previous result failed this is the wrong value at current_filter_method so it might be matching here for the wrong filter types

Reply:
changed to if filter_method in ('PCF', 'PCF+ESM')